google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataSettings.java

```diff
@@ -275,6 +275,15 @@ public CredentialsProvider getCredentialsProvider() {
       return stubSettings.getCredentialsProvider();
     }
 
+    /**
+     * Sets to true to enable channels to periodically reconnect to GFE, false (default) to disable
+     */
```

Comment:
Customers don't really know what a GFE is, so I would say "Cloud Bigtable servers". Also I would expand the docs to something like:

```
Configure periodic gRPC channel refreshes.

<p>This feature will gracefully refresh connections to the Cloud Bigtable service. This is an experimental feature to address tail latency caused by the service dropping long lived gRPC connections, which causes the client to renegotiate the gRPC connection in the request path, which causes periodic spikes in latency 

```